src/test/java/org/cactoos/collection/CollectionOfTest.java

```diff
@@ -79,4 +87,132 @@ public void testToStringEmpty() {
         ).affirm();
     }
 
+    @Test()
+    public void returnsIteratorWithSupportedRemove() {
+        final CollectionEnvelope<String> list = new CollectionEnvelope<String>(
+            new CollectionOf<>("eleven")
+        ) { };
+        final Iterator<String> iterator = list.iterator();
+        iterator.next();
+        iterator.remove();
+        new Assertion<>(
+            "Iterator should be empty",
```

Comment:
@victornoel please, change `"Iterator should be empty"`  to `"Must be Iterator empty"`.

Reply:
@fabriciofx IMHO, in this case it does not sound properly in English. The correct form is:"Iterator must be empty" or "Must be empty"(if iterator is meant).

Reply:
@fanifieiev is "Must be an empty Iterator" better?

Reply:
@fabriciofx Well, yes, it is better, but still the meaning is not as MUST if you want to express obligation, but rather is a certainty.
You can have a look at the way 'must' is used at https://www.grammar.cl/english/must.htm or at https://www.crownacademyenglish.com/modal-verb-must-form-use-meaning/
So, what you suggest is called "Deductions and conclusions"

Reply:
@fanifieiev so, what's your recommendation? Please, note (almost) all fail test messages in Cactoos starts with Must...

Reply:
@fabriciofx Well, it is not a matter of whether they start with 'Must' or not.
For instance: 
- "Must keep unique integer numbers" is correct as an obligation;

Well it usually the combination of 'must' and 'be', or sometimes 'must' and 'have+<perfect tense>' which usually means "Deductions and conclusions", but not always(English is fun:))

So, my recommendation is to properly use the combination of "must" and "be".

More examples:
- "It must be cold outside" (not an obligation, but means 'It seems cold outside')

Reply:
@fabriciofx "Iterator must be empty" is a good example.
As an alternative, "Must return an empty Iterator" is also good.

Reply:
@fanifieiev I liked "Must return an empty Iterator". Thanks a lot @fanifieiev!!! ✌️ 😄

Reply:
@fabriciofx You are welcome!

Reply:
@fabriciofx @fanifieiev thx, I've applied this for all the other below too

Reply:
@victornoel ok